scripts/Jenkinsfile_batch_test

```diff
@@ -50,7 +50,7 @@ node {
               def job_test_args = "${TEST_ARGS}"
               if ( test_name == "upgrademanual" || test_name == "upgradereplacement" || test_name == "upgradesuc"){
                 actual_testdir_name = "upgradecluster"
-                if ( test_name == "sucupgrade") {
+                if ( test_name == "upgradesuc") {
                   job_test_args = job_test_args + " -tags=${test_name} -sucUpgradeVersion ${VERSION_OR_COMMIT}"
```

Comment:
shouldnt we do something like this here? 
```
              if ( test_name == "upgrademanual" || test_name == "upgradereplacement" || test_name == "upgradesuc"){
                actual_testdir_name = "upgradecluster"

                if ( test_name == "upgradesuc") {
                  job_test_args = job_test_args + " -tags=${test_name} -sucUpgradeVersion ${VERSION_OR_COMMIT}"
                } else {
                  job_test_args = job_test_args + " -tags=${test_name} -installVersionOrCommit ${VERSION_OR_COMMIT}"
                }

                  job_test_args = job_test_args +  " -channel ${UPGRADE_CHANNEL} "
              }
```

Since in all we use channel and avoid code duplication

Reply:
since the  upgrade flag is the same for manual and replacement

Reply:
fixed it. the reason i did it though is right now only manual upgrade jenkins job was using it. the rest did not use a channel flag in the test args. as long as the go test takes the channel flag, there should not be a problem.